src/hydra/hydra-mercure.js

```diff
@@ -43,16 +43,28 @@ export const on = (mercure, topics, callback) => {
     topics = [topics];
   }
   const wrapper = (event) => {
+    let item;
+    try {
+      item = JSON.parse(event.data);
+    } catch (e) {
+      console.debug(e);
+      return;
+    }
+
+    if ('object' !== typeof item) {
```

Comment:
Do we no longer accept IRIs?

Reply:
API-Platform always sends plain objects through Mercure, not just IRIs.